notesapplication/notes/api_v1/serializers.py

```diff
@@ -26,3 +57,49 @@ def create(self, validated_data):
         )
         note.save()
         return note
+
+    def update(self, instance, validated_data):
+        request = self.context.get("request", None)
+        note_version = NoteVersion.objects.create(
+            note_id=instance,
+            edited_by=request.user,
+            title=instance.title,
+            text=instance.text,
+            date_updated=date.today(),
```

Comment:
You don't have to add today's date if you correctly set up the `updated_date`.
```
date_created = models.DateField(default=str(date.today()))
date_updated = models.DateField(default="")
```
This is wrong. Try searching for the Django way of setting DateTime fields.